openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -173,6 +173,9 @@
   align-items: center;
   justify-content: center;
 }
+.flex-shrink {
+  flex-shrink: 0;
+}
```

Comment:
Let's have this in `position.less`

All the position related CSS should be there: display, position, flex, align, justify, etc